Twig/Extension.php

```diff
@@ -107,8 +107,17 @@ public function piwikCode()
     private function addDefaultApiCalls()
     {
         $this->paqs[] = ['enableLinkTracking'];
-
-        foreach ($this->paqs as $paq) {
+        $lastPaq = [];
```

Comment:
Could you rename this variable to make its intention clearer? e. g. $manuallyAddedTrackPageView

Reply:
I would also suggest to initialize it as `null`